apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,24 +1,48 @@
 import { NavLink } from "react-router-dom";
 
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
 import styles from "./styles.module.css";
 
+type ButtonVariant = "danger" | "default" | "outlined";
+
 type Properties = {
 	href?: string | undefined;
+	isDisabled?: boolean;
 	label: string;
+	onClick?: () => void;
 	type?: "button" | "submit";
+	variant?: ButtonVariant;
 };
```

Comment:
What do you think about adding a width property to the Button component? For example, in my task, I need a modal with two buttons of the same width. Or is there a better way to achieve this?

Reply:
Check this [comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/issues/87#issuecomment-2316084322) and its replies